test/jdk/java/awt/color/ICC_Profile/ValidateICCHeaderData/ValidateICCHeaderData.java

```diff
@@ -1,4 +1,4 @@
-/*
+package ValidateICCHeaderData;/*
```

Comment:
What's going on here ?

Reply:
It was removed in subsequent commit. You might want to refresh the page.